master/static/srv/get_model_version.sql

```diff
@@ -27,6 +27,7 @@ c AS (
     COALESCE(c.framework, '') as framework,
     COALESCE(c.format, '') as format,
     COALESCE(c.determined_version, '') as determined_version,
+    (v.metrics->'validation_metrics'->>(e.config->'searcher'->>'metric'))::float8 AS searcher_metric,
```

Comment:
tests found these were missing, @ioga of some interest maybe

Reply:
cc @mapmeld

Reply:
these and other recent issues feel like strong justification for using an orm, or at least something that autogens types from sql queries. (sqlc)

Reply:
I'd be amazed if sqlc is capable of correctly processing these

Reply:
Yeah, same. I took a look at their issues and it looks like it's far from perfect. But still is cool.